plain-text.ts

```diff
@@ -0,0 +1,10 @@
+export default {
```

Comment:
We need to select a i18n plugin and then start doing this but for now just add this text onto the component itself

Reply:
Ok!